Samples/kotlin/src/main/java/com/phunware/kotlin/sample/NavigationOverlayView.kt

```diff
@@ -89,13 +90,28 @@ class NavigationOverlayView @JvmOverloads constructor(context: Context, attrs: A
     private lateinit var navigator: Navigator
     private lateinit var adapter: ManeuverPagerAdapter
 
+    /**
+     * Since this custom view extends a ViewPager, we need to handle "click listening" on our own.
+     */
+    private var clickListener: OnClickListener? = null
+
+    init {
+        // set the viewpager to show multiple pages at once, so maneuver steps peak in from the side
+        offscreenPageLimit = 6
+        pageMargin = (-40).toPx()
+    }
+
     private val pageChangeListener = object : ViewPager.SimpleOnPageChangeListener() {
         override fun onPageSelected(position: Int) {
             val pair = adapter.getItem(position)
             navigator.setCurrentManeuver(pair.mainPos)
         }
     }
 
+    override fun setOnClickListener(l: OnClickListener?) {
```

Comment:
nitpicky, but do we want a more descriptive parameter name than "l" like listener?

Reply:
looks good